make/examples.mk

```diff
@@ -1,7 +1,6 @@
 # Copyright 2023 D2iQ, Inc. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-
 KUBERNETES_VERSION ?= v1.27.5
```

Comment:
Lets either remove this `KUBERNETES_VERSION ` var or use it in the kustomize

Reply:
I want to use it! But how?